examples/google-cloud-speech/output/google-cloud-speech/lib/google/cloud/speech.rb

```diff
@@ -0,0 +1,125 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+require "google/gax"
+require "pathname"
+
+module Google
+  module Cloud
+    ##
+    # # Ruby Client for Cloud Speech API
+    #
+    # [Cloud Speech API][Product Documentation]:
+    # Converts audio to text by applying powerful neural network models.
+    # - [Product Documentation][]
+    #
+    # ### Installation
+    #
+    # ```
+    # $ gem install google-cloud-speech
+    # ```
+    #
+    # ### Preview
+    #
+    # #### SpeechClient
+    #
+    # ```rb
+    # require "google/cloud/speech"
+    #
+    # speech_client = Google::Cloud::Speech.new
+    # language_code = "en-US"
+    # sample_rate_hertz = 44100
+    # encoding = :FLAC
+    # config = {
+    #   language_code: language_code,
+    #   sample_rate_hertz: sample_rate_hertz,
+    #   encoding: encoding
+    # }
+    # uri = "gs://bucket-name/hello.flac"
+    # audio = { uri: uri }
+    # response = speech_client.recognize(config, audio)
+    # ```
+    #
+    # ### Next Steps
+    #
+    # - Read the [Cloud Speech API Product documentation][Product Documentation]
+    #   to learn more about the product and see How-to Guides.
+    # - View this [repository's main README](https://github.com/googleapis/google-cloud-ruby/blob/master/README.md)
+    #   to see the full list of Cloud APIs that we cover.
+    #
+    # [Product Documentation]: https://cloud.google.com/speech
+    #
+    module Speech
+      FILE_DIR = File.realdirpath Pathname.new(__FILE__).join("..").join("speech")
+
+      AVAILABLE_VERSIONS = Dir["#{FILE_DIR}/*"].select { |file| File.directory?(file) }
+                                               .select { |dir| Google::Gax::VERSION_MATCHER.match(File.basename(dir)) }
+                                               .select { |dir| File.exist?(dir + ".rb") }
+                                               .map { |dir| File.basename(dir) }
+
+      ##
+      # Service that implements Google Cloud Speech API.
+      #
+      # @param version [Symbol, String]
+      #   The major version of the service to be used. By default :v1
+      #   is used.
+      # @overload new(version:, credentials:, scopes:, client_config:, timeout:)
```

Comment:
Missing `metadata:` and `exception_transformer:`.

Reply:
Also, why are `interceptors` not an argument?